Resources/public/js/views/ezsearch-searchview.js

```diff
@@ -0,0 +1,64 @@
+YUI.add('ezsearch-searchview', function (Y) {
+    Y.namespace('eZSearch');
+
+    Y.eZSearch.SearchView = Y.Base.create('ezsearchSearchView', Y.eZ.TemplateBasedView, [], {
+        events: {
+            '.ezsearch-form': {
+                'submit': '_handleFormSubmit'
+            },
+        },
+
+        initializer: function () {
+            this.after('searchRequestChange', function() {
+                this.fire('searchRequest', {
+                    searchString: this.get('searchRequest'),
+                });
+            })
+        },
+
+        render: function () {
+            this.get('container').setHTML(
+                this.template({
+                    "inputString": this.get("searchString"),
+                })
+            );
+            this._uiSetMinHeight();
+            return this;
+        },
+
+        /**
+         * Handles the `submitForm` event by preventing the original form to be
+         * submitted by the browser and by changing searchRequest attribute.
+         *
+         * @method _handleFormSubmit
+         * @protected
+         * @param {EventFacade} e
+         */
+        _handleFormSubmit: function (e) {
+            var form = e.currentTarget,
+                searchString = form.get('searchstring').get('value');
+
+            e.preventDefault();
+            this._set('searchRequest', searchString);
```

Comment:
if the attribute is not read only, there's no point in using `_set`, you can use `set`.